### PR TITLE
Collapse elements even if the page is refreshed

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
https://www.loom.com/share/37e4b22e916c4befb7eb322b5051662d

### CODE REVIEW

**Comment by owocki** [comment]
@zoek1 looks like u left some commits in here u didnt intend to?

**Comment by owocki** [comment]
here is the spec i gave zoek @gitcoinco/engineers 

```
hey zoek; the town square used to remember when you had collapsed a module, and remember that that module was collapsed for next time you came to the town square.  it would just set a localstorge flag for each module. somwehwere along the way, this functionality got removed.

this was really important for mobile usability

any chance youd be open to re-adding it
``z`

**Comment by zoek1** [comment]
Bas branch @owocki :see_no_evil:

File: app/assets/v2/js/pages/townsquare.js [hunk 1]
```diff
@@ -143,6 +152,15 @@ $(document).ready(function() {
       if (!hide_promo) {
         $('.promo').removeClass('hidden');
       }
+
+      $('.townsquare_block-header').each(function() {
+        let target_id = $(this).data('target');
+        var item = localStorage.getItem(target_id);
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/townsquare.js:158]: const

File: app/assets/v2/js/pages/townsquare.js [hunk 2]
```diff
@@ -46,6 +46,15 @@ $(document).ready(function() {
     }
   });
 
+  // collapse menu items
+  $('body').on('click', '.townsquare_block-header', function(e) {
+    let target_id = $(this).data('target');
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/townsquare.js:51]: const

**Comment by thelostone-mc** [comment]
Closing this out cause looks like we got this cherry-picked :P

**Pull request closed without merging**